sdk/cosmosdb/cosmos/test/public/common/TestHelpers.ts

```diff
@@ -77,6 +77,19 @@ export async function getTestContainer(
   return db.container(id);
 }
 
+export async function getmaxIntegratedCacheStalenessTestContainer(
```

Comment:
Am not seeing the caller, can you please help me point to the callers?

Reply:
This is a "helper" function. The caller was in a previous commit, and that code is no longer needed, but it's good to keep a helper function.

Reply:
removed